Chess-FromZeroToHero.Contracts/Dtos/User/UserWithIdDto.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Chess_FromZeroToHero.Contracts.Dtos.Base;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chess_FromZeroToHero.Contracts.Dtos.User
+{
+    public class UserWithIdDto : BaseWithIdDto
+    {
+        public string Username { get; set; }
+
+        public string Password { get; set; }
+
+        public int Rating { get; set; }
+
+        public string Name { get; set; }
+
+        public int Age { get; set; }
+
+        public byte[] ProfilePicture { get; set; }
```

Comment:
Let's leave storing profile pictures as another feature we can implement later, because we won't be storing them in the DB.